front/src/modules/ui/board/components/BoardColumn.tsx

```diff
@@ -60,6 +60,20 @@ const StyledAmount = styled.div`
   margin-left: ${({ theme }) => theme.spacing(2)};
 `;
 
+const StyledNumChildren = styled.div`
+  background-color: ${({ theme }) => theme.background.tertiary};
+  border-radius: ${({ theme }) => theme.border.radius.rounded};
+  color: ${({ theme }) => theme.font.color.tertiary};
+  line-height: ${({ theme }) => theme.text.lineHeight.lg};
+  margin-left: auto;
+  padding-bottom: ${({ theme }) => theme.spacing(1 / 2)};
```

Comment:
let's actually use a height with justify-content: center + align-items: center actually.
using half padding seems hacky here :)

Reply:
Do you want me to remove the padding? Also adding height and justify-content: center + align-items doesn't seem to do anything. Maybe I've misunderstood what you mean.

Reply:
Done!